source/plugin/modularization.rst

```diff
@@ -0,0 +1,61 @@
+===================
+Code Modularization
+===================
+
+When writing Sponge plugins, you may find that you need to modularize your code. Maybe you want to split your core into
+another project so that others can make use of it, or maybe you want to utilize a service another plugin provides. With
+Sponge, gone are the days of shading entire plugins into your plugin.
```

Comment:
If I recall correctly, you did, but I'll change it.


Get Outlook for Android



From: Ryan Leach

Sent: Saturday, July 16, 9:19 PM

Subject: Re: [SpongePowered/SpongeDocs] Add Code Modularization (#516)

To: SpongePowered/SpongeDocs

Cc: Meyer Zinn, Author



In source/plugin/modularization.rst:


> @@ -0,0 +1,61 @@ > +=================== > +Code Modularization > +=================== > + > +When writing Sponge plugins, you may find that you need to modularize your code. Maybe you want to split your core into > +another project so that others can make use of it, or maybe you want to utilize a service another plugin provides. With > +Sponge, gone are the days of shading entire plugins into your plugin. 


You didn't need to do this on Bukkit either, people just did so they didn't run into versioning issues, cut the snark.


—

You are receiving this because you authored the thread.

Reply to this email directly, view it on GitHub, or mute the thread.